app/assets/javascripts/maps.es6

```diff
@@ -38,9 +38,13 @@ window.COPO.maps = {
 
     function getCheckinData(page) {
       if ($('.c-devices.a-show').length !== 0) {
-        return $.getJSON(`${window.location.href}/checkins?page=${page}&per_page=1000`)
+        if(window.location.search.length !== 0){
```

Comment:
spaces are missed here

__need to review all js code in this PR for styling issues__